RELEASE.md

```diff
@@ -1,9 +1,15 @@
 To release a new version of the widgets on PyPI:
 
-1. Update \_version.py
+1. Update \_version.py (set release version, remove 'dev')
 2. `git add` and `git commit`
 3. `python setup.py sdist upload` 
 4. `python setup.py bdist_wheel upload`
```

Comment:
Yeah, you can upload as many times as you want (as long as it is a different file)